Control/Monad/Logic.hs

```diff
@@ -159,6 +161,20 @@ observeManyT n m
 runLogicT :: LogicT m a -> (a -> m r -> m r) -> m r -> m r
 runLogicT (LogicT r) = r
 
+-- | Convert from 'LogicT' to an arbitrary logic monad transformer.
```

Comment:
Could you please mention examples of other logic monad transformers here? Linking `logict-sequence` is fine.